webcrawler.py

```diff
@@ -5,12 +5,21 @@
 import scrapy
 from scrapy.crawler import CrawlerProcess
 from scrapy.http.request import Request
+import re
+
 
 KEYWORDS_FILE = 'textfile.txt'
 SCRAPY_OUTPUT_FILE = 'scrapy-output.json'
 RANKED_OUTPUT_FILE = 'output.csv'
 
 
+def contains_date(url):
+    result= re.findall(r'/(\d{4})/(\d{1,2})/(\d{1,2})/', url)
+    if (len(result) == 0):
+        return False
+    else: return True
+# A function that returns a boolean based upon if a url contains a date; this is used to filter non articles out
```

Comment:
This comment should be the first line inside the function. Or better yet, write it as a [docstring](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings):

```python
def contains_date(url):
    """Returns True if the url contains a date."""
```